composer.json

```diff
@@ -32,11 +32,11 @@
     "ckeditor/panelbutton": "^4.10",
     "composer/installers": "^1.2",
     "cweagans/composer-patches": "^1.6.5",
-    "drupal-composer/drupal-scaffold": "^2.5",
     "drupal/better_normalizers": "^1.0@beta",
     "drupal/components": "^1",
     "drupal/config_installer": "^1",
-    "drupal/core": "^8.7.5",
+    "drupal/core-composer-scaffold": "^8.8.0",
+    "drupal/core-recommended": "^8.8.0",
```

Comment:
Not sure this change needed, looks better to stick to core

Reply:
Working like a charm atm, `make tests` passing successfully